jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -39,6 +39,64 @@
       :hint="viewer_multiselect ? 'Select viewers to set options simultaneously' : 'Select the viewer to set options.'"
     />
 
+    <v-row v-if="viewer_selected === 'spectrum-viewer' || (viewer_selected[0] === 'spectrum-viewer' && viewer_selected.length === 1)">

```

Comment:
this logic prevents this from being used in lcviz (and will conflict with any future support for renaming viewers)

Reply:
does the logic have to be in python with an extra traitlet or can we just use the existing logic in the `PlotOptionsSyncState` by doing `v-if="viewer_x_min_sync.in_subscribed_states"`?

Reply:
I don't really know the details of what subscribed states is doing, so: maybe? 😆 

I'll give it a try.

Reply:
Yeah, that doesn't stop the viewer bounds menu from showing up if you multiselect a spectrum viewer and an image viewer. So I think the way I have it now is necessary.